dune-project

```diff
@@ -1,2 +1,5 @@
-(lang dune 1.0)
+(lang dune 3.0)
 (name tls)
+(formatting disabled)
+(using mdx 0.2)
+(homepage https://github.com/mirleft/ocaml-tls)
```

Comment:
What purpose does this homepage have here?

Reply:
I don't remember why I added it. I removed it and the CI is still happy, so I guess it's not needed.